tools/jsonschema_to_rust_dto_enum_flags.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+import os
+import re
+import sys
+import json
+
+from typing import Iterator, cast
+
+DEFAULT_INDENT = 4
+DOCS_URL = 'https://github.com/kamu-data/open-data-fabric/blob/master/open-data-fabric.md#{}-schema'
+PREAMBLE = """
+///////////////////////////////////////////////////////////////////////////////
+// WARNING: This file is auto-generated from Open Data Fabric Schemas
+// See: http://opendatafabric.org/
+///////////////////////////////////////////////////////////////////////////////
+
+use bitflags::bitflags;
+
+use crate::MetadataEvent;
+
+///////////////////////////////////////////////////////////////////////////////
+"""
+
+JsonType = dict[str, 'JsonType']
+SchemaName = str
+
+
+def render(schemas_dir: str) -> None:
+    print(PREAMBLE)
+
+    name, sch = read_schema(schemas_dir, 'metadata-events/MetadataEvent.json')
+
+    try:
+        print('/' * 80)
+        print(f'// {name}')
+        print('// ' + DOCS_URL.format(name.lower()))
+        print('/' * 80)
+        print()
+
+        for l in render_schema(name, sch):
+            print(l)
+        print()
+
+
+    except Exception as ex:
+        raise Exception(
+            f'Error while rendering {name} schema:\n{sch}'
+        ) from ex
+
+
+def read_schema(schemas_dir: str, file_path: str) -> (SchemaName, JsonType):
```

Comment:
Can you change this to use `utils/schemas.py`? It has an identical function.

I started introducing common utils module, but did not yet update some codegens to use it. Sorry for the mess.

Reply:
Done... and it's OK, these are helper scripts, so there's no problem :)